server/proxy/bindings.go

```diff
@@ -195,7 +195,11 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 		}
 
 		if b.Form != nil {
-			cleanForm(*b.Form)
+			clean, problems := cleanForm(*b.Form)
+			for _, prob := range problems {
+				log.WithError(prob).Debugf("invalid form field in bingding")
```

Comment:
Maybe we want to combine these into one log message? It could get pretty spammy in the logs as separate messages, though it's DEBUG level so I guess that's to be expected.

Reply:
Yeah, debug's fine IMO

Reply:
```suggestion
				log.WithError(prob).Debugf("invalid form field in binding")
```